src/js/utils.js

```diff
@@ -33,6 +33,7 @@ class Utils {
 		}
 
 		const toggleEls = this.headerEl.querySelectorAll('[data-o-header-togglable]');
+		this.toggleHandler = this.toggleHandler.bind(this);
```

Comment:
Other handlers needed it because they needed to reference the utils instance, but the toggleHandler doesn't use this anywhere